.coveragerc

```diff
@@ -0,0 +1,24 @@
+[run]
+source = django_geosource
+omit =
+    django_geosource/migrations/*
+    django_geosource/apps.py
```

Comment:
best practice is to define 'default_app_config = 'django_geosource.apps.DjangoGeosourceConfig' in django_geosource.__init__, apps. will be hit at each app initialisation

Reply:
https://docs.djangoproject.com/fr/2.2/ref/applications/#configuring-applications